components/cmp/services/src/useCases/index.js

```diff
@@ -0,0 +1,11 @@
+import {GetConsentStatus} from './getConsentStatus'
```

Comment:
Why are you using useCases folder instead if UseCases and sendConsents instead of SendConsentsUseCase for instance.

Reply:
I don't get this comment, I think is missing some words.

Reply:
I'm talking about the naming of the files. If it's a pseudo domain it should applied domain's naming convention.